### PR TITLE
Closes #2429 UI update for "Combine Google Fonts files" feature

### LINKED ISSUE #2429: UI update for "Combine Google Fonts files" feature
During the discussion for #2241 we had the following feedback: 

- https://github.com/wp-media/wp-rocket/issues/2241#issuecomment-586631339
^ Lucy said: 
```
To be a bit pedantic, the option is called "Combine Google Fonts" so it makes logical sense that if there isn't anything to combine (i.e there isn't more than 1 font), nothing is going to be done. I would propose that we should rename the option "Optimize Google Fonts" then it will make sense to process a single font and it takes into account the fact that we're doing more than just combining.
```

- https://github.com/wp-media/wp-rocket/issues/2241#issuecomment-586666669
^ Vasilis said:
```
There is confusion about this among some of our customers who think that we combine Google Fonts. There is no way for us to either combine or optimize the actual fonts. We optimize Google fonts' requests.

On top of what @webtrainingwheels suggested, I'd propose we call it "Optimize Google Fonts' requests".
```

Based on this we had a discussion for new UI:

- https://wp-media.slack.com/archives/CFSCRLVGB/p1583496051012400
- Jonathan also suggested that we could think about removing this option and making it automatic - https://wp-media.slack.com/archives/CFSCRLVGB/p1584019694023000?thread_ts=1583505331.015200&cid=CFSCRLVGB

### PR DESCRIPTION
**Pull request by crystinutzaa**
Changed text:
Option Title: Optimize Google Fonts (Current: Combine Google Fonts files)
Option Description: Improves font performance and combines multiple font requests to reduce the number of HTTP requests (Current: Combining Google Fonts will reduce the number of HTTP requests.)

URL for the doc ^ will be: `https://docs.wp-rocket.me/article/1312-optimize-google-fonts`
And the article ID: `5e8687c22c7d3a7e9aea4c4a`

Closes #2429 

TODO:
- [x] - New architecture

### CODE REVIEW

File: inc/Engine/Admin/Beacon/Subscriber.php [hunk 1]
```diff
@@ -1,16 +1,15 @@
 <?php
```
Comment (Tabrisrp) [inline inc/Engine/Admin/Beacon/Subscriber.php:1]: You can delete this file now, it's no longer used

File: inc/Engine/Admin/ServiceProvider.php [hunk 2]
```diff
@@ -37,17 +36,19 @@ class Admin_Subscribers extends AbstractServiceProvider {
...
 		$this->getContainer()->add( 'deactivation_intent_render', 'WP_Rocket\Admin\Deactivation\Render' )
 			->withArgument( $this->getContainer()->get( 'template_path' ) . '/deactivation-intent' );
 		$this->getContainer()->share( 'deactivation_intent_subscriber', 'WP_Rocket\Subscriber\Admin\Deactivation\Deactivation_Intent_Subscriber' )
 			->withArgument( $this->getContainer()->get( 'deactivation_intent_render' ) )
 			->withArgument( $this->getContainer()->get( 'options_api' ) )
-			->withArgument( $this->getContainer()->get( 'options' ) );
-		$this->getContainer()->share( 'beacon_subscriber', 'WP_Rocket\Subscriber\Admin\Settings\Beacon_Subscriber' )
+			->withArgument( $options );
+		$this->getContainer()->share( 'beacon_subscriber', 'WP_Rocket\Engine\Admin\Beacon\Subscriber' )
 			->withArgument( $this->getContainer()->get( 'beacon' ) );
```
Comment (Tabrisrp) [inline inc/Engine/Admin/ServiceProvider.php:50]:
```suggestion
```

**Comment by hellofromtonya** [comment]
@arunbasillal This PR is ready for QA.

**Comment by arunbasillal** [comment]
Starting QA

**Pull request merged**